lib/technical_metadata_service.rb

```diff
@@ -30,13 +30,12 @@ def add_update_technical_metadata
     old_techmd = old_technical_metadata
     new_techmd = new_technical_metadata(deltas)
     # this is version 1 or previous technical metadata was not saved
-    return store(new_techmd) if old_techmd.nil?
+    return new_techmd if old_techmd.nil?
```

Comment:
Docs above need to change if method signature is changing.

Reply:
Thanks! I've updated the return value for `.add_update_technical_metadata`